client/src/Pages/productpage/productpage-styles.js

```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components";
```

Comment:
For the product page, the original layout displayed one products per row, whereas I styled it to display 4 per row.